security/permissions.rst

```diff
@@ -103,3 +103,10 @@ Policy Creation Example:
 .. note::
 
     Multiple AccessPolicy resources containing the same Reference will be combined. In the above example if the user Alice is found in another policy with ``user/Patient.c``, the resulting permission will be ``user/Patient.crs``
+
+Search capabilities:
+^^^^^^^^^^^^^^^^^^^^
+The `restriction` for `type.code = "smart-v2"` support search capabilites, and placeholders within that search statement. The placeholder will have the following format: `#placeholder#`, and will be replaced by a claim with the same name of the placeholder that is provided in the authorization token. If the placeholder claim is not provided, it will result in a unauthorized result. Example restrictions:
```

Comment:
I think you meant to use monospace, which requires double backticks.
```suggestion
The ``restriction`` for ``type.code = "smart-v2"`` support search capabilites, and placeholders within that search statement. The placeholder will have the following format: ``#placeholder#``, and will be replaced by a claim with the same name of the placeholder that is provided in the authorization token. If the placeholder claim is not provided, it will result in a unauthorized result. Example restrictions:
```